integration/spark/src/main/scala/org/apache/spark/sql/secondaryindex/util/SecondaryIndexUtil.scala

```diff
@@ -310,20 +284,22 @@ object SecondaryIndexUtil {
   /**
    * This method deletes the old index files or merge index file after data files merge
    */
-  private def deleteOldIndexOrMergeIndexFiles(
+  private def collectOldIndexOrMergeIndexFiles(
       factTimeStamp: Long,
       validSegments: util.List[Segment],
       indexCarbonTable: CarbonTable): Unit = {
+    val oldIndexAndMergeIndexFiles = new util.ArrayList[String]
     // delete the index/merge index carbonFile of old data files
```

Comment:
update this comment as now we arent deleting

Reply:
done